src/controller/python/test_client/Dockerfile

```diff
@@ -1,5 +1,4 @@
-# getting base image for ubuntu
-FROM ubuntu:20.10
+FROM ubuntu:latest
```

Comment:
Is this going to be "OK" going forward? Or should we lock it to 21.04?

Reply:
Im not sure yet, maybe lock it down once 21.04 becomes LTS version? This one is just the test image, the TH image would be more crucial